jetty-server/src/main/java/org/eclipse/jetty/server/session/Session.java

```diff
@@ -792,7 +764,7 @@ public void renewId(HttpServletRequest request)
                     case CHANGING:
                         try
                         {
-                            _stateChangeCompleted.await();
```

Comment:
I actually preferred this the way it was:   the stateChangeCompleted condition makes it very clear what is being awaited and released